.github/workflows/build_and_deploy.yml

```diff
@@ -8,7 +8,7 @@ on:
 env:
   NAPI_CLI_VERSION: 2.16.2
   TURBO_VERSION: 1.10.9
-  NODE_LTS_VERSION: 20
+  NODE_LTS_VERSION: 21
```

Comment:
Nit: 21 is not the LTS version, should we rename the variable to avoid ambiguity?